packages/bits/demo/src/components/demo/wizard-v2/wizard-restore-state/wizard-restore-state.example.component.html

```diff
@@ -0,0 +1,187 @@
+    </nui-dialog-header>
+    <form class="dialog-body compact-mode"
+          [formGroup]="form"
+          *ngIf="form">
+        <nui-wizard-horizontal
+            linear
+            [state]="state"
+            (finished)="saveState($event)"
+        >
+            <nui-wizard-step-v2 label="Person Details"
+                                formGroupName="personDetails"
+                                [stepControl]="form.get(['personDetails'])">
+                <nui-form-field [control]="form.get(['personDetails','name'])"
+                                class="d-block mb-4 mt-4"
+                                i18n-caption
+                                caption="Name">
+                    <nui-textbox formControlName="name"></nui-textbox>
+                    <nui-validation-message for="required"
+                                            *ngIf="form.get(['personDetails','name'])?.errors?.required"
+                                            i18n>
+                        This is required
+                    </nui-validation-message>
+                </nui-form-field>
+
+                <nui-form-field [control]="form.get(['personDetails','privacy'])">
+                    <nui-checkbox formControlName="privacy" [value]="true">
+                        Privacy Policy
+                    </nui-checkbox>
+
+                    <nui-validation-message for="required"
+                                            *ngIf="form.get(['personDetails','privacy'])?.errors?.required"
+                                            i18n>
+                        This is required
+                    </nui-validation-message>
+                </nui-form-field>
+
+                <ng-template wizardStepFooter>
+                    <ng-container *ngTemplateOutlet="wizardCancel"></ng-container>
+                    <ng-container *ngTemplateOutlet="wizardNext; context: { form: 'personDetails' }"></ng-container>
+                </ng-template>
+            </nui-wizard-step-v2>
+
+            <nui-wizard-step-v2 label="Organization"
+                                formGroupName="organization"
+                                [stepControl]="form.get(['organization'])">
+                <nui-form-field [control]="form.get(['organization', 'title'])"
+                                class="d-block my-4"
+                                i18n-caption
+                                caption="Organization">
+                    <nui-textbox formControlName="title"></nui-textbox>
+                    <nui-validation-message for="required"
+                                            *ngIf="form.get(['organization','title'])?.errors?.required"
+                                            i18n>
+                        This is required
+                    </nui-validation-message>
+                </nui-form-field>
+                <nui-form-field caption="Date"
+                                class="d-block my-4"
```

Comment:
The form fields in the old wizard examples just used `mb-4`, so it seems like this might add too much space between the form fields when they're right next to each other. Here's a form field in the old wizard docs: https://github.com/solarwinds/nova/blob/98ef229973f8b1903efa77c902b2cc2c2f5cc9b1/packages/bits/demo/src/components/demo/wizard/wizard-reset-step/wizard-reset-step.example.component.html#L33

Reply:
fixed